subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/declaring_repositories.adoc

```diff
@@ -1,837 +1,346 @@
+Common use cases include:
 
-- performance, when you know a dependency will never be found in a specific repository
-- security, by avoiding leaking what dependencies are used in a private project
-- reliability, when some repositories contain corrupted metadata or artifacts
+* performance (e.g. you know a dependency will never be found in a specific repository)
+* security  (e.g. avoiding leaking dependencies used in a private project)
+* reliability (e.g. some repositories contain corrupted metadata or artifacts)
 
-It's even more important when considering that the declared order of repositories matter.
+To add a filter to a repository, add an include or exclude clause to the `content` block within the repository configuration.
+The following example:
 
-[[sec:declaring-repository-filter]]
-=== Declaring a repository filter
+* uses an inclusion clause to fetch only dependencies produced by a specific organization from a maven repository defined with a custom URL
+* uses an exclusion clause to exclude dependencies produced by that same organization from the Maven Central repository defined with built-in shorthand notation
 
-.Declaring repository contents
 ====
 include::sample[dir="snippets/dependencyManagement/declaringRepositories-filtering/groovy",files="build.gradle[tags=repository-filter]"]
 include::sample[dir="snippets//dependencyManagement/declaringRepositories-filtering/kotlin",files="build.gradle.kts[tags=repository-filter]"]
 ====
 
-By default, repositories include everything and exclude nothing:
+By default, repositories include everything and exclude nothing.
+Adding an include or exclude clause has the following side effects:
 
-* If you declare an include, then it excludes everything _but_ what is included.
-* If you declare an exclude, then it includes everything _but_ what is excluded.
-* If you declare both includes and excludes, then it includes only what is explicitly included and not excluded.
+* If you declare only includes, Gradle includes *only dependencies that match the includes*.
+* If you declare only excludes, Gradle includes *everything except dependencies that match the excludes*.
+* If you declare both includes and excludes, Gradle includes *only dependencies that match the includes that also do not match the excludes*.
 
-It is possible to filter either by explicit _group_, _module_ or _version_, either strictly or using regular expressions.
-When using a strict version, it is possible to use a version range, using <<single_versions.adoc#single-version-declarations,the format supported>> by Gradle.
-In addition, there are filtering options by resolution context: configuration name or even configuration attributes.
+You can filter by _group_, _module_ or _version_, either with strict string matching or regular expressions.
+You can filter versions by version range, using <<single_versions.adoc#single-version-declarations,the Gradle version format>>.
+You can also filter by configuration name or attributes.
```

Comment:
thought: I do not like the "you can" pattern here. Less words are better and wonder if this would be best in a list format?

Reply:
Fixed: https://github.com/gradle/gradle/pull/22943/files#r1038471110